### ISSUE TITLE
Currently unable to build on windows

### ISSUE THREAD

**Issue by photex** [LoopPerfect/buckaroo#67]
Heyo!

I'm new to buck and buckaroo so bear with me.

Starting by following the instructions for building buckaroo on your homepage I get the following result:
`BUILD FAILED: Must specify at least one build target`

Reading over the BUCK file I realize I can try again and specify //:buckaroo-cli. Great stuff starts happening but then quickly fails with the following error:

```
C:\projects\buckaroo>..\buck\bin\buck build //:buckaroo-cli
[-] PARSING BUCK FILES...FINISHED 0.0s

C:\projects\buckaroo\buck-out\gen\jgit-jar-fixed__srcs>cp buck-out/gen/jgit-jar/jgit-4.5.0.jar C:\projects\buckaroo\buck-out\gen\jgit-jar-fixed\jgit-4.5.0-fixed.jar   && zip -d C:\projects\buckaroo\buck-out\gen\jgit-jar-fixed\jgit-4.5.0-fixed.jar META-INF/*.RSA META-INF/*.SF META-INF/*.MF

'cp' is not recognized as an internal or external command,
operable program or batch file.

BUILD FAILED: //:jgit-jar-fixed failed with exit code 1:
genrule
stderr: 'cp' is not recognized as an internal or external command,
operable program or batch file.

[-] BUILDING...FINISHED 0.5s [100%] (24/27 JOBS, 1 UPDATED, 1 [3.7%] CACHE MISS)
[-] DOWNLOADING... (0.00 B/S AVG, TOTAL: 0.00 B, 0 Artifacts)
```

**Comment by njlr** [comment]
Ah, that is definitely a bug on our end! 

It looks like we wrote part of the build script for macOS / Linux using `cp` but didn't write the version for Windows cmd. 

I will take a look. 

------------

In the meantime, you might find it helpful to look at this example of Buck on Windows: [https://github.com/ilya-klyuchnikov/buck-windows-cxx](https://github.com/ilya-klyuchnikov/buck-windows-cxx). 

This should be enough to get up and running with Buck while we fix Buckaroo.

**Comment by njlr** [comment]
If you are using Git Bash on Windows, then the current install procedure should work: 

```bash=
buck build :buckaroo-cli
java -jar ./buck-out/gen/buckaroo-cli.jar
```

**Comment by njlr** [comment]
The [latest commit](https://github.com/LoopPerfect/buckaroo/commit/6f9a3cd906bc66d0055ec8e6f46e4c9bdbfa2a25) should fix things for PowerShell. 

Let me know how you get on!

**Comment by photex** [comment]
Running it now, it can't find the `zip` command. Tried to run from powershell but I get the error that I haven't enabled running scripts... I'll look into that now. This marks the first time I've bothered to even run powershell haha.

**Comment by njlr** [comment]
In PowerShell try:

choco install zip

(Assuming you have Chocolatey installed from when you installed Buck)

On 6 Jun 2017, at 20:44, Chip Collier <notifications@github.com<mailto:notifications@github.com>> wrote:


Running it now, it can't find the zip command. Tried to run from powershell but I get the error that I haven't enabled running scripts... I'll look into that now. This marks the first time I've bothered to even run powershell haha.
